egs/wsj/s5/steps/cleanup/internal/modify_ctm_edits.py

```diff
@@ -373,7 +373,10 @@ def PrintNonScoredStats():
     num_incorrect_lines = num_lines - num_correct_lines
     percent_lines_incorrect= '%.2f' % (num_incorrect_lines * 100.0 / num_lines)
     percent_modified = '%.2f' % (num_lines_modified * 100.0 / num_lines);
-    percent_of_incorrect_modified = '%.2f' % (num_lines_modified * 100.0 / num_incorrect_lines)
+    if num_incorrect_lines>0:
```

Comment:
can you please ensure there are spaces around the > operator?  per pep8 style or whatever it's called.
and break this line after the '/' as the line is getting very long.

Reply:
while you're at it, why not make the fix to that bugfix so @vimalmanohar doesn't have to?